OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs

```diff
@@ -64,7 +64,7 @@ public void StartGame(Arguments args)
 			var connect = args != null ? args.GetValue("Launch.Connect", null) : null;
 			if (!string.IsNullOrEmpty(connect))
 			{
-				var parts = connect.Split(':');
+				var parts = connect.Replace("openra://", "").Split(':');
```

Comment:
`Launch.Connect` is defined to take a `hostname:port` argument.  Changing it to accept `Launch.Connect=openra://hostname:port` so that you don't need to do any validation under linux is most certainly a platform-specific workaround.